app-dicts/myspell-de/myspell-de-2017.01.12.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+MYSPELL_DICT=(
+	"de_AT.aff"
+	"de_AT.dic"
+	"de_DE.aff"
+	"de_DE.dic"
+	"de_CH.aff"
+	"de_CH.dic"
+)
+
+MYSPELL_HYPH=(
+	"hyph_de_AT.dic"
+	"hyph_de_DE.dic"
+	"hyph_de_CH.dic"
+)
+
+MYSPELL_THES=(
+	"th_de_AT_v2.dat"
+	"th_de_AT_v2.idx"
+	"th_de_DE_v2.dat"
+	"th_de_DE_v2.idx"
+	"th_de_CH_v2.dat"
+	"th_de_CH_v2.idx"
+)
+
+inherit myspell-r2
+
+DESCRIPTION="German (AT,CH,DE) dictionaries for myspell/hunspell"
+HOMEPAGE="
+	https://extensions.libreoffice.org/extensions/german-de-at-frami-dictionaries
+	https://extensions.libreoffice.org/extensions/german-de-ch-frami-dictionaries
+	https://extensions.libreoffice.org/extensions/german-de-de-frami-dictionaries
+"
+SRC_URI="
+	https://extensions.libreoffice.org/extensions/german-de-at-frami-dictionaries/$(ver_rs 1 -)/@@download/file/dict-de_AT-frami_$(ver_rs 1- -).oxt
```

Comment:
`$(ver_rs 1- -)` looks like a good candidate for putting into `MY_PV` to avoid repetition.

Reply:
Done. Also dropped IUSE and added default to src_prepare().